.github/workflows/prod-workflow.yml

```
@@ -122,3 +122,60 @@ jobs:
+        with:
+          components: clippy, rustfmt
+          target: wasm32-unknown-unknown
+          toolchain: stable
+
+      - name: Install dependencies
+        run: |
+          cargo install toml-cli
+
+      - name: Cache builder
+        id: fetcher-build-cache
+        uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)